### PR TITLE
TAJO-1856: Add a description about the relationship of tablespace, managed table, and external table to Tablespace section of Table Management chapter

### PR DESCRIPTION
**Pull request by eminency**
Hi @hyunsik , @jihoonson .

One more PR, please review.

Thanks.

### CODE REVIEW

File: tajo-docs/src/main/sphinx/table_management/tablespaces.rst [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
+Once defined, a tablespace can be referred to by name when creating a database or a table.
+Especially, it is very useful when a Tajo cluster instance should use heterogeneous storage systems such as HDFS, MySQL, and Oracle.
 
+============================================
+External Table, Managed Table and Tablespace
+============================================
+
+Tajo has two types of table. One is external table. It needs **location** property when the table is created. Using this property, you can create an external table indicating existing external data source.
+For example, if there is already your data as Text/JSON files or HBase table, you can register it as tajo external table.
+Other one is managed table, which means internal table, that is created in Tajo internal table space. You can convert external data to the form that you want and save it in managed table.
```
Comment (jihoonson) [inline tajo-docs/src/main/sphinx/table_management/tablespaces.rst]: Tablespace is a single word.

Comment (jihoonson) [inline tajo-docs/src/main/sphinx/table_management/tablespaces.rst]: The definition of managed tables is the tables which are created and managed by Tajo. One of their characteristics is that they are stored in directories under Tajo's root directory.

File: tajo-docs/src/main/sphinx/table_management/tablespaces.rst [hunk 2]
```diff
@@ -1,9 +1,26 @@
...
+
+Tablespace is used for managed tables and it is a kind of alias represents physical(distributed, usually) storages. When managed table is created, you can specify a tablespace with **tablespace** keyword, or default tablespace will be used.
```
Comment (jihoonson) [inline tajo-docs/src/main/sphinx/table_management/tablespaces.rst]: I suggest ```Tablespace is a predefined physical location where data stored on. It is supported for only managed tables. When you create a managed table, you can use the **tablespace** keyword to specify the location of data will be stored. If the tablespace is not specified, the default tablespace of the table's database is used.```

File: tajo-docs/src/main/sphinx/table_management/tablespaces.rst (hunk 1 above)
Comment (jihoonson) [inline tajo-docs/src/main/sphinx/table_management/tablespaces.rst]: I made some mistakes. Tablespace's characteristic is each tablespace has a physical location and every data in a tablespace is stored there. Tajo's root directory on hdfs is an example physical locatuon of hdfs tablespace.

**Comment by eminency** [comment]
Refined more simply, please check.

**Comment by jihoonson** [comment]
+1 LGTM.

**Pull request closed without merging**